tree/30_generic_methods/service_check_started_at_boot.cf

```diff
@@ -29,16 +29,16 @@ bundle agent service_check_started_at_boot(service_name)
 {
   vars:
 
-    pass2.systemctl_utility_present.!broken_systemctl::
+    pass1.systemctl_utility_present.!broken_systemctl::
       "command_to_check"   string => "${paths.path[systemctl]} is-enabled ${service_name}.service";
 
-    pass2.!systemctl_utility_present.chkconfig_utility_present::
+    pass1.!systemctl_utility_present.chkconfig_utility_present::
       "command_to_check"   string => "${paths.path[chkconfig]} --list ${service_name} | grep -q -e 3:on -e B:on";
 
-    pass2.!systemctl_utility_present.!chkconfig_utility_present.lsitab_utility_present::
+    pass1.!systemctl_utility_present.!chkconfig_utility_present.lsitab_utility_present::
       "command_to_check"   string => "/usr/sbin/lsitab -a | egrep '^${service_name}:[0-9]+:(respawn|boot|bootwait|wait|once|initdefault|sysinit):'";
 
-    pass2.(!(systemctl_utility_present|chkconfig_utility_present|lsitab_utility_present)|broken_systemctl)::
+    pass1.(!(systemctl_utility_present|chkconfig_utility_present|lsitab_utility_present)|broken_systemctl)::
```

Comment:
We should have a coherent writing of class context to make it more readable.
So I'd prefer the form: 
pass2.!systemctl_utility_present.!chkconfig_utility_present.!lsitab_utility_present